forge-gui/res/cardsfolder/upcoming/sarevok_deathbringer.txt

```diff
@@ -0,0 +1,9 @@
+Name:Sarevok, Deathbringer
+ManaCost:3 B
+Types:Legendary Creature Human Knight
+PT:3/4
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ Player | Revolt$ None | Execute$ TrigLoseLife | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of each player's end step, if no permanents left the battlefield this turn, that player loses X life, where X is NICKNAME's power.
```

Comment:
Is `Revolt$ None` fine or should we use a more generic param since Revolt is not mentioned on the card?

Reply:
I did use `Revolt$ True` on Lulu in #650 as well fwiw

Reply:
Looks alright to me :)